src/YoutubeAtomPlayer.tsx

```diff
@@ -32,6 +32,9 @@ declare global {
 
 type YoutubeCallback = (e: YT.PlayerEvent & YT.OnStateChangeEvent) => void;
 
+type PlayCustomEventDetail = { videoId: string };
```

Comment:
Sorry, nits again, but could this be `CustomPlayEventDetail`?

Reply:
done 👍